src/Components/HookForm/HookForm.tsx

```diff
@@ -0,0 +1,253 @@
+        )}
+      </label>
+      <label htmlFor="password">
+        <div className={classes.spaceBetween}>
+          <div>Password</div>
+          <div className={classes.passwordStrenthBlock}>
+            Strength:
+            <div
+              className={classes.strengthIndicator}
+              style={{ background: strengthIndicatorClass }}
+            ></div>
+            <div className={classes.strengthStatus}>{strengthStatus}</div>
+          </div>
+        </div>
+        <input
+          type="password"
+          className={classes.formInput}
+          id="password"
+          {...register('password')}
+          onChange={(e) => changeStrengthIndicator(e.target.value)}
+        />
+        {errors.password ? (
+          <div className={classes.errorMessage}>{errors.password.message}</div>
+        ) : (
+          <span className={classes.errorMessage}></span>
+        )}
+      </label>
+      <label htmlFor="confirmPassword">
+        Confirm password:
+        <input
+          type="password"
+          className={classes.formInput}
+          id="confirmPassword"
+          {...register('confirmPassword')}
+        />
+        {errors.confirmPassword ? (
+          <span className={classes.errorMessage}>
+            {errors.confirmPassword.message}
+          </span>
+        ) : (
+          <span className={classes.errorMessage}></span>
+        )}
+      </label>
+      <label htmlFor="gender">
+        Gender:
+        <select
+          className={classes.formSelect}
+          id="gender"
+          {...register('gender')}
+        >
+          <option value="Male">Male</option>
+          <option value="Female">Female</option>
+        </select>
+        {errors.gender ? (
+          <span className={classes.errorMessage}>{errors.gender.message}</span>
+        ) : (
+          <span className={classes.errorMessage}></span>
+        )}
+      </label>
+      <label htmlFor="country">
+        Country:
+        <input
+          className={classes.formInput}
+          list="countries"
+          id="country"
+          {...register('country')}
+        />
+        <datalist id="countries">
+          {countriesList.map((item: CountriesListItem) => {
+            return <option key={item.code} value={item.name} />;
+          })}
+        </datalist>
+        {errors.country ? (
+          <span className={classes.errorMessage}>{errors.country.message}</span>
+        ) : (
+          <span className={classes.errorMessage}></span>
+        )}
+      </label>
+      <label htmlFor="image">
+        Image
+        <input
+          id="image"
+          type="file"
+          {...register('image')}
+          onChange={(e) => {
+            setUploadedFile(e);
+          }}
+        />
+        {imageUrl ? (
+          <img
+            className={classes.formImage}
+            src={imageUrl}
+            alt="image"
+            {...register('image')}
+          />
+        ) : (
+          ''
+        )}
```

Comment:
1. don't need to cover `''` into `()`;
2. better to use `&&` in condition or return `null` if it false